content/python/concepts/time-module/sleep.md

```diff
@@ -0,0 +1,46 @@
+---
+
+Title: '.sleep()'
+
+Description: 'The function suspends execution of current thread for a given time.'
+
+Subject: 'Computer Science'
+
+Tags: 
+    -'Python'
+    -'Function'
+    -'Time Module'
+
+CatalogContent:
+    - 'learn-python'
+    - 'concepts/time-module'
+---
+
+# **.sleep()**
+The python time module function .sleep() suspends execution of current thread for a given period of time(in seconds).
+
+### **Syntax** 
+`time.sleep(23.4);` 
```

Comment:
This line should be in a pseudo code block, Also, it should be more general, like `time.sleep(duration)` with the description explaining what `duration` is.